pom.xml

```diff
@@ -178,6 +229,16 @@
                     </compilerArgs>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Are you sure we need this?

Reply:
this plugin was not required, removed it. Thanks